public/apps/configuration/constants.tsx

```diff
@@ -240,6 +240,7 @@ export const CLUSTER_PERMISSIONS: string[] = [
   'indices:data/read/msearch/template',
   'indices:data/read/mtv',
   'indices:data/read/mtv*',
+  'indices:data/read/search/template/render',
```

Comment:
Is there a way we can add a test that will tell us if these lists are out of date - such as with this action?

Reply:
It is possible to get a full list of transport actions that have been registered in core (native actions + transport actions), but there are a couple of problems:

1. The actions aren't necessarily categorized the same way that the security plugin categorizes the actions. i.e. This new action is a _cluster_ permission even though it begins with `indices:`. At some point, I think these need to be revisited to ensure that the action naming is consistent and works with the security plugin. 
2. Not all transport actions are meant to be run by users, some may be internal and only invoked by a process. I think the security plugin's [ConfigUpdateAction](https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/action/configupdate/ConfigUpdateAction.java#L34) is one example of that.

Reply:
Actually - we've already got an issue around this [1], not a deal breaker for this change :D  

@cwperks  Would you mind (re)posting your thoughts on the issue?

- [1] https://github.com/opensearch-project/security-dashboards-plugin/issues/1196